CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 - Handles cases where certain control flow type expressions have comments between patterns/keywords and the pattern ident contains the keyword [#5009](https://github.com/rust-lang/rustfmt/issues/5009)
 - Handles tuple structs that have explicit visibilities and start with a block style comment [#5011](https://github.com/rust-lang/rustfmt/issues/5011)
 - Handles leading line-style comments in certain types of macro calls [#4615](https://github.com/rust-lang/rustfmt/issues/4615)
+- Fixes bizarre string indentation in macro defs with `format_strings` [#4036](https://github.com/rust-lang/rustfmt/issues/4036)
```

Comment:
I appreciate you thinking to include an entry in the changelog! However, we'll need to remove, or move+change this.

The section you've added this to is actually for the already-released 1.4.38 version, so we shouldn't add notes for this fix to a version where it doesn't exist. Secondly, we don't want to use vague, subjective language and adjectives like those typically provided by frustrated users in issue reports. Users reading this type of line in a changelog can't really glean much information from it, which is instead why try to frame the wording in our changelog in more of a functional manner that describes the behavior that was fixed in a way anyone could understand (e.g. `Fixed issue in macro defs where every line but the first line in a multiline string was missing an indent level when ``format_strings`` was enabled`).

If you'd like to keep an entry (which would indeed be helpful for me whenever it comes time for the next release), let's adjust the wording a bit and move to the top under the `Unreleased` section of the page

Reply:
No problem. I thought I had placed it under `Unreleased`, but I obviously wasn't paying attention.